### PR TITLE
Testing API SQL response 

### PR DESCRIPTION
**Pull request by unrealwork**
Tested :
- response codes
- response headers

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/sql/SqlApiResponseTests.java [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+ */
+public class SqlApiResponseTests extends BaseMethod {
+    private static final String METHOD_SQL_API = "/api/sql";
+    private static WebTarget httpSqlApiResource = httpRootResource.path(METHOD_SQL_API);
+    private static final String ALLOW_METHODS = "Access-Control-Allow-Methods";
+
+
+    @Test
+    public void testAllowMethods() {
+        String expectedAllowMethods = "HEAD, GET, POST, PUT, PATCH, DELETE";
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/SqlApiResponseTests.java]: I think we should use `final` for this case

File: src/test/java/com/axibase/tsd/api/method/sql/SqlApiResponseTests.java [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+/**
+ * @author Igor Shmagrinskiy
+ */
+public class SqlApiResponseTests extends BaseMethod {
+    private static final String METHOD_SQL_API = "/api/sql";
+    private static WebTarget httpSqlApiResource = httpRootResource.path(METHOD_SQL_API);
+    private static final String ALLOW_METHODS = "Access-Control-Allow-Methods";
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/SqlApiResponseTests.java]: ALLOW_METHODS is used once. Should we declare it as class field?

If so, why we do not declare here `Content-type`? We use it 3 times

**Comment by Sild** [comment]
OK

**Comment by Sild** [comment]
still ok

**Comment by rmakulov** [comment]
Ok.

File: src/test/java/com/axibase/tsd/api/method/sql/response/SqlApiResponseCodesTests.java [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    @BeforeClass
+    public static void createTestData() throws InterruptedException, JSONException, IOException {
+        final Sample[] testSamples = {
+                new Sample("2016-06-03T09:23:00.000Z", "16.0"),
+                new Sample("2016-06-03T09:26:00.000Z", "8.1"),
+                new Sample("2016-06-03T09:36:00.000Z", "6.0"),
+                new Sample("2016-06-03T09:41:00.000Z", "19.0")
+        };
+        for (Sample s : testSamples) {
+            testSeries.addData(s);
```
Comment (KutuzovGregory) [inline src/test/java/com/axibase/tsd/api/method/sql/response/SqlApiResponseCodesTests.java]: inline this to testSeries.addData(new Sample(...)) or testSeries.addData(LIST)

File: src/test/java/com/axibase/tsd/api/method/sql/response/SqlApiResponseCodesTests.java [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+        }
+        if (!SeriesMethod.insertSeries(testSeries, 1000)) {
```
Comment (KutuzovGregory) [inline src/test/java/com/axibase/tsd/api/method/sql/response/SqlApiResponseCodesTests.java]: introduce boolean variable here

File: src/test/java/com/axibase/tsd/api/method/sql/response/SqlApiResponseHeadersTests.java [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    @BeforeClass
+    public static void createTestData() throws InterruptedException, JSONException, IOException {
+        final Sample[] testSamples = {
+                new Sample("2016-06-03T09:23:00.000Z", "16.0"),
+                new Sample("2016-06-03T09:26:00.000Z", "8.1"),
+                new Sample("2016-06-03T09:36:00.000Z", "6.0"),
+                new Sample("2016-06-03T09:41:00.000Z", "19.0")
+        };
+        for (Sample s : testSamples) {
+            testSeries.addData(s);
```
Comment (KutuzovGregory) [inline src/test/java/com/axibase/tsd/api/method/sql/response/SqlApiResponseHeadersTests.java]: the same as above

File: src/test/java/com/axibase/tsd/api/method/sql/response/SqlApiResponseHeadersTests.java [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+        }
+        SeriesMethod.insertSeries(testSeries, 1000);
+    }
+
+
+    @Test
+    public void testAllowMethods() {
+        final String expectedAllowMethods = "HEAD, GET, POST, PUT, PATCH, DELETE";
```
Comment (KutuzovGregory) [inline src/test/java/com/axibase/tsd/api/method/sql/response/SqlApiResponseHeadersTests.java]: should they be in exactly this order?

**Comment by KutuzovGregory** [comment]
ok

**Pull request merged**